test/plugin/scenarios/apm-toolkit-trace-scenario/configuration.yml

```diff
@@ -18,3 +18,8 @@ type: jvm
 entryService: http://localhost:8080/apm-toolkit-trace-scenario/case/tool-kit
 healthCheck: http://localhost:8080/apm-toolkit-trace-scenario/case/healthCheck
 startScript: ./bin/startup.sh
+extendEntryHeader:
```

Comment:
Is this config for injecting headers?

Reply:
Yes, follow this config I can read the extension header.